js/bitfinex2.js

```diff
@@ -11,7 +11,7 @@ module.exports = class bitfinex2 extends bitfinex {
     describe () {
         return this.deepExtend (super.describe (), {
             'id': 'bitfinex2',
-            'name': 'Bitfinex',
+            'name': 'Bitfinex v2',
```

Comment:
The `v2` was removed on purpose: https://github.com/ccxt/ccxt/issues/4658

Reply:
Oops, that was not intentional.

Reply:
@npomfret no worries )